app/Product/Aspect/ExampleProductAspect.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Example\Project\Product\Aspect;
+
+use Go\Aop\Aspect;
```

Comment:
ah then it is like the serializer 🥇it also can be a pain to figure out what is going on/wrong